avocado/utils/pci.py

```diff
@@ -135,7 +141,8 @@ def get_slot_from_sysfs(full_pci_address):
 
     :return: slot of PCI address from sysfs.
     """
-    if not os.path.isfile('/sys/bus/pci/devices/%s/devspec' % full_pci_address):
+    if not os.path.isfile('/sys/bus/pci/devices/%s/devspec' %
+                          full_pci_address):
```

Comment:
I am seeing below pep8 error so I am forced to make this change in this patch.

pci.py:158:80: E501 line too long (80 > 79 characters)